ast/src/gen/generic.rs

```diff
@@ -2679,7 +2679,7 @@ impl<R> Node for Excepthandler<R> {
 }
 
 #[derive(Clone, Debug, PartialEq)]
-pub struct Arguments<R = TextRange> {
+pub struct PythonArguments<R = TextRange> {
```

Comment:
Are the `PythonArgument`s still used? If so, when do we use which/How does the parser know which structure it should generate?